### ISSUE TITLE
create_cluster_firmware_upgrade_item doesn't work with the simultaneous option

### ISSUE THREAD

**Issue by howardfeiock** [Isilon/isilon_sdk_8_0_python#56]
There's a mismatch in the UpgradeApi->create_cluster_firmware_upgrade_item method when specifying the simultaneous option.  According to the docs, this is expecting a bool value.  When running the method with simultaneous=True, I get the following output:

```
Exception when calling UpgradeApi->create_cluster_firmware_upgrade_item: (400)
Reason: Bad Request
HTTP response headers: HTTPHeaderDict({'Date': 'Wed, 24 Aug 2022 16:56:42 GMT', 'Server': 'Apache', 'Allow': 'POST, HEAD', 'X-Frame-Options': 'sameorigin', 'Strict-Transport-Security': 'max-age=31536000;', 'Connection': 'close', 'Transfer-Encoding': 'chunked', 'Content-Type': 'application/json'})
HTTP response body: 
{
"errors" : 
[

{
"code" : "AEC_BAD_REQUEST",
"field" : "simultaneous",
"message" : "Field: simultaneous has error: A property was present that is not defined by the schema."
},

{
"code" : "AEC_BAD_REQUEST",
"message" : "Input validation failed."
}
]
}
```

I tried using the REST API semantics of upgrade_type="simultaneous", but that's blocked by the definition of ClusterFirmwareUpgradeItem:

```
TypeError: ClusterFirmwareUpgradeItem.__init__() got an unexpected keyword argument 'upgrade_type'
```

I'm using the 9.1.0 version of the module.

**Comment by dhanush03122003** [comment]
HI @howardfeiock  , 
Can you check if the same issue is on the latest version link : https://github.com/Isilon/isilon_sdk_python , version : Isilon_SDK_v0.3.0.1 .

**Comment by dhanush03122003** [comment]
Here is the link for latest version : https://pypi.org/project/isilon-sdk/ , version : Isilon_SDK_v0.4.0.

**Comment by mmanwani07** [comment]
Since we did not hear back on the issue, and multiple release already done on top of it, we ae closing the issue.
